config/charts/inferencepool/templates/epp-deployment.yaml

```diff
@@ -27,6 +27,45 @@ spec:
       # Conservatively, this timeout should mirror the longest grace period of the pods within the pool
       terminationGracePeriodSeconds: 130
       containers:
+      {{- if .Values.inferenceExtension.sidecar.enabled }}
+      - name: {{ .Values.inferenceExtension.sidecar.name }}
+        image: {{ .Values.inferenceExtension.sidecar.image }}
+        imagePullPolicy: {{ .Values.inferenceExtension.sidecar.image.pullPolicy | default "Always" }}
```

Comment:
I recommend `IfNotPresent` default

Reply:
done. Also updated the epp imagepullpolicy to make it consistent